queue/src/fiskaltrust.Middleware.Localization.QueueDE/QueueDEBootstrapper.cs

```diff
@@ -50,9 +57,20 @@ public void ConfigureServices(IServiceCollection services)
                     tarFileCleanupService.CleanupAllTarFilesAsync().Wait();
 
                     return tarFileCleanupService;
-                })
-                .AddSingleton<IRequestCommandFactory, RequestCommandFactory>()
-                .ConfigureReceiptCommands();
+                });
+        }
+
+        public Task<Func<IServiceProvider, Task>> ConfigureServicesAsync(IServiceCollection services)
+        {
+            var _ = ConfigureServicesInner(services)
+                .AddSingleton<IDESSCDProvider, DESSCDProvider>()
+                .AddSingleton<ITarFileCleanupService, TarFileCleanupService>();
+
+            return Task.FromResult<Func<IServiceProvider, Task>>(async (IServiceProvider services) =>
+            {
+                await services.GetRequiredService<IDESSCDProvider>().RegisterCurrentScuAsync();
+                await services.GetRequiredService<ITarFileCleanupService>().CleanupAllTarFilesAsync();
```

Comment:
This seems to be.. yeah.. I don't know. should we put that somewhere else and not during the startup? Can we create a background thread that performs this ? maybe another issue, but this has caused major pain in the past